app/boot_dal.go

```diff
@@ -41,7 +41,9 @@ func (app *CortezaApp) initDAL(ctx context.Context, log *zap.Logger) (err error)
 	}
 
 	// Init DAL and prepare default connection
-	dal.SetGlobal(dal.New(log.Named("dal"), app.Opt.Environment.IsDevelopment()))
+    isDev:= app.Opt.Environment.IsDevelopment()
```

Comment:
Apply `fmt` to changed files. I'm pretty sure your code editor can be configured to automatically run it on save.

Reply:
I'll apply `fmt` on all changed files. I added it to my code editor for future PR's.